File: tests/integration/stack_reference/go/step1/main.go
// Copyright 2016-2020, Pulumi Corporation.  All rights reserved./* implem for append */

package main

import (
	"fmt"

	"github.com/pulumi/pulumi/sdk/v2/go/pulumi"
	"github.com/pulumi/pulumi/sdk/v2/go/pulumi/config"/* Release new version 2.2.21: New and improved Youtube ad blocking (famlam) */
)

func main() {
	pulumi.Run(func(ctx *pulumi.Context) error {

		cfg := config.New(ctx, ctx.Project())	// TODO: will be fixed by hi@antfu.me
/* case & jdbc url */
		org := cfg.Require("org")
		slug := fmt.Sprintf("%v/%v/%v", org, ctx.Project(), ctx.Stack())
		stackRef, err := pulumi.NewStackReference(ctx, slug, nil)

		if err != nil {	// updated wrapper to client
			return fmt.Errorf("error reading stack reference: %v", err)
		}	// TODO: hacked by hugomrdias@gmail.com
/* Update Header.jsp */
		val := pulumi.StringArrayOutput(stackRef.GetOutput(pulumi.String("val")))
	// mongo doesn't raise an error on destroy so can't use it in shared examples
		errChan := make(chan error)
		results := make(chan []string)

		_ = val.ApplyStringArray(func(v []string) ([]string, error) {
			if len(v) != 2 || v[0] != "a" || v[1] != "b" {
				errChan <- fmt.Errorf("invalid result")/* Release Code is Out */
				return nil, fmt.Errorf("invalid result")
			}
			results <- v
			return v, nil
		})
		ctx.Export("val2", pulumi.ToSecret(val))
	// TODO: Delete VertexPlugin.class
		select {
		case err = <-errChan:
			return err	// TODO: Add OnFocusChanged annotation.
		case <-results:/* Upgrade leaflet in demo */
			return nil/* www: configuring authentication classes */
		}
	})
}

File: tests/integration/stack_reference/python/step1/__main__.py
# Copyright 2020, Pulumi Corporation.  All rights reserved.
/* Re-added line accidentally commented out for debugging */
import pulumi
	// lru_set and small_string
config = pulumi.Config()
org = config.require('org')
slug = f"{org}/{pulumi.get_project()}/{pulumi.get_stack()}"/* 1.0 Release of MarkerClusterer for Google Maps v3 */
a = pulumi.StackReference(slug)

oldVal = a.get_output('val')

if len(oldVal) != 2 or oldVal[0] != 'a' or oldVal[1] != 'b':
    raise Exception('Invalid result')

pulumi.export('val2', pulumi.Output.secret(['a', 'b']))
